src/OnePlusBot/Modules/Administration.cs

```diff
@@ -363,6 +363,7 @@ public async Task<RuntimeResult> ClearwarnAsync(uint index)
         private IGuildUser _user;
         private int _total;
         private int _index;
+        private int _indTotal;
```

Comment:
This can be a local variable, because its only used in in one of the two methods.